File: DataSummitSolution/DataSummitUI/src/app/profileVersion/create/models/rectanglepairs.ts
import { RectanglePair } from "./rectanglepair"

export class RectanglePairs {

    CompanyId: number
    Height: number
    Width: number
    Pairs: RectanglePair[]
    TemplateName: string

    constructor(companyId?: number, height?: number, 
        width?: number, rectanglePairs?: RectanglePair[],
        templateName?: string) {
            this.CompanyId = companyId || 0
            this.Height = height || 0
            this.Width = width || 0 
            this.Pairs = rectanglePairs
            this.TemplateName = templateName || ''
    }
}

File: DataSummitSolution/DataSummitWeb/DTO/RectanglePairs.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DataSummitWeb.DTO
{
    public class RectanglePairs
    {
        public int CompanyId { get; set; }
        public long Height { get; set; }
        public long Width { get; set; }
        public string TemplateName { get; set; }
        public List<RectanglePair> Pairs { get; set; }

        public RectanglePairs()
        {
            Pairs = new List<RectanglePair>();
        }
    }
}
